data/battle_anim_scripts.s

```diff
@@ -13979,7 +13979,7 @@ Move_RISING_VOLTAGE::
 	end @to do:
 
 Move_TERRAIN_PULSE::
-	end @to do:
+	goto Move_DRAGON_PULSE @Temporary, needs to setup different animations based on terrain
```

Comment:
What does the animation look like in game? A simple pulse animation that changes particle color based on terrain would be relatively easy to implement